src/main/java/core/basesyntax/service/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(Path filePath) {
+        List<String> records;
+        if (filePath.toFile().exists()) {
```

Comment:
what do you return if the file doesn’t exist? Better remove it and get an exception in the catch block